### DISCUSSION TITLE
Is there any way to guide not to detect certain defect?

### DISCUSSION THREAD

**Discussion by papago2355** [open-edge-platform/anomalib#1399] [Q&A] [answered]
Hi. I am working on detecting anomaly in Plastic mold.
Currently it performs really well, but I have one issue.
This Plastic mold is combined with a metal based lid.
I want to detect only plastic part not a metal part.
Is there anyway to guide my model not to detect metal part?
Currently using PADIM/segmentation.

PS. can self made anomaly masks solve my issue?

**Comment by blaz-r** [comment, marked as answer]
Most models are unsupervised, they don't really use a ground truth mask when training. This means that even if you have a mask that you created, it's not going to be used in training. In case of Padim, the model is trained by calculating a distribution for each patch position in image for all normal images in set. So in this case, mask are ignored when training. 

If you can have position of metallic part fixed or somehow first detect it, I think one way of handling this is to use regular image processing to mask that part out.
